server.tf

```diff
@@ -7,9 +11,19 @@ resource "aws_security_group_rule" "server-ingress" {
   security_group_id        = aws_security_group.server.id
 }
 
+resource "aws_security_group_rule" "server-ingress-extra" {
+  count             = length(var.cidr_blocks_extra) > 0 ? 1 : 0
```

Comment:
Please put a newline after count per the Terraform [style guide](https://www.terraform.io/docs/configuration/style.html).